doozer/doozerlib/cli/get_nightlies.py

```diff
@@ -243,6 +269,16 @@ def rc_api_url(tag: str, arch: str) -> str:
     """
     arch = go_arch_for_brew_arch(arch)
     arch_suffix = go_suffix_for_arch(arch)
```

Comment:
I noticed that we have private bool in the func which we can use
```suggestion
    arch_suffix = go_suffix_for_arch(arch, private_nightly)
```

Reply:
so the base URL for private nightlies is a bit different. from non-amd64, the arch is in the URL, but for amd64, its not there, so have that if else to substitute it anyways. i.e. for amd64, we can't use the arch_suffix.

tldr, even if we use the private bool function, we can update only one lie that is:
```
# For other arches, add it to the URL
        return f"{constants.RC_BASE_PRIV_URL.format(arch=arch)}/api/v1/releasestream/{tag}{arch_suffix}-priv"
```
to
```
# For other arches, add it to the URL
        return f"{constants.RC_BASE_PRIV_URL.format(arch=arch)}/api/v1/releasestream/{tag}{arch_suffix}"
```

Reply:
done